app/src/test/java/ca/llamabagel/transpo/ui/trips/GetNextBusTripsUseCaseTest.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2019 Derek Ellis. Subject to the MIT license.
+ */
+
+package ca.llamabagel.transpo.ui.trips
+
+import ca.llamabagel.transpo.data.TestStops
+import ca.llamabagel.transpo.data.TripsRepository
+import ca.llamabagel.transpo.data.provideFakeTripsRepository
+import ca.llamabagel.transpo.ui.trips.adapter.TripItem
+import ca.llamabagel.transpo.utils.provideFakeCoroutinesDispatcherProvider
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.flow.singleOrNull
+import kotlinx.coroutines.runBlocking
+import org.junit.Assert.*
+import org.junit.Test
+
+class GetNextBusTripsUseCaseTest {
+
+    private val repository = provideFakeTripsRepository()
+    private val getNextBusTrips = GetNextBusTripsUseCase(repository, provideFakeCoroutinesDispatcherProvider())
+
+    @Test
+    fun `get bus trips simple`() = runBlocking {
+        val flow = getNextBusTrips(TestStops.walkleyJasper.id)
+
+        repository.getTrips(TestStops.walkleyJasper.id)
+
+        val trips = flow.first()
+        assertEquals(3, trips.size)
```

Comment:
Wouldn't it be closer to english to have the literal on the right? ie. `trips.size == 3` instead of `3 == trips.size`.

Reply:
the params are labeled `expected, actual`. They need to be in that order in case the assert fails and it displays a message.